### PR TITLE
Adding support for VeteranRecordService

### PR DESCRIPTION
**Pull request by bwisnaskas-va**
Support for VeteranRecordService has been added. This will be used by lighthouse-benefits-letters.

Three methods have been implemented:
findVeteranByPtcpntId
findCorporateRecordByFileNumber
findBirlsRecordByFileNumber

### CODE REVIEW

File: bgs-model/pom.xml [hunk 1]
```diff
@@ -57,6 +57,7 @@
                 <wsdlFile>share-standard-data/shareStandardDataWebService.wsdl</wsdlFile>
                 <wsdlFile>standard-data/standardDataService.wsdl</wsdlFile>
                 <wsdlFile>svn-type-v2/svnTypeV2Service.wsdl</wsdlFile>
+                <wsdlFile>VeteranRecordService/VetRecordWebService.wsdl</wsdlFile>
```
Comment (esubocz-va) [inline bgs-model/pom.xml:60]: super tiny, but we should try to follow naming conventions for the directory and wsdl names. directories `-` and wsdl names start with lowercase.

File: bgs-model/src/main/resources/META-INF/wsdl/VeteranRecordService/VetRecordWebService.wsdl [hunk 2]
```diff
@@ -0,0 +1,299 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Published by JAX-WS RI at http://jax-ws.dev.java.net. RI's version is Oracle JAX-WS 2.1.5. -->
+<!-- Generated by JAX-WS RI at http://jax-ws.dev.java.net. RI's version is Oracle JAX-WS 2.1.5. -->
+<definitions xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/" xmlns:tns="http://services.share.benefits.vba.va.gov/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://schemas.xmlsoap.org/wsdl/" targetNamespace="http://services.share.benefits.vba.va.gov/" name="VetRecordWebService">
+    <types>
+        <xsd:schema>
+            <xsd:import namespace="http://services.share.benefits.vba.va.gov/" schemaLocation="VetRecordWebService.xsd"/>
```
Comment (esubocz-va) [inline bgs-model/src/main/resources/META-INF/wsdl/VeteranRecordService/VetRecordWebService.wsdl:7]: XSD file name should be updated for convention as well, and will need to be edited here.

Comment (bwisnaskas-va) [reply bgs-model/src/main/resources/META-INF/wsdl/VeteranRecordService/VetRecordWebService.wsdl:7]: So for the XSD, the change will be the same as the wsdl, just change to lowercase first letter?

File: bgs-model/pom.xml (hunk 1 above)
Comment (bwisnaskas-va) [reply bgs-model/pom.xml:60]: Ok. I will update that.

**Review by esubocz-va — APPROVED** [review]

**Pull request merged**